.circleci/config.yml

```diff
@@ -3,10 +3,12 @@ version: 2
 # https://circleci.com/blog/circleci-hacks-reuse-yaml-in-your-circleci-config-with-yaml/
 defaults: &defaults
   docker:
-    - image: docker.io/weaveworks/wksctl-build:switch-away-from-quay-53c3c712-WIP
+    - image: docker.io/weaveworks/wksctl-build:update-build-image-cb52a6ea
       environment:
         GOPATH: /go/
         SRCDIR: /src/github.com/weaveworks/wksctl
+        KUBECTL_URL: https://dl.k8s.io/v1.10.5/kubernetes-client-linux-amd64.tar.gz
```

Comment:
Could we upgrade this one? I'd be surprised if it didn't cause strangeness in our tests.

Reply:
Sure, but in a different PR.  That's matching the existing code.